bin/ci-builder

```diff
@@ -157,6 +180,15 @@ case "$cmd" in
             --env NIGHTLY_CANARY_CONFLUENT_CLOUD_API_SECRET
             --env PYPI_TOKEN
         )
+
+        if [[ $detach_container == "true" ]]; then
+            args+=("--detach")
+        fi
+
+        if [[ -n "$container_name_param" ]]; then
+            args+=("--name=$container_name_param")
```

Comment:
I'm a bit confused why you had to handle the `--detach` and `--name` and then build them back into the same strings that they were before. Wouldn't the `$@` below take care of that as long as they are in the correct order?

Reply:
`--detach` and `--name` need to run into `args`, which the rest needs to into `docker_command`. The latter ones need to come after the docker image definition (see line 267).